packages/git-utils/tests/main.mjs

```diff
@@ -1,10 +1,8 @@
-'use strict'
```

Comment:
This file must be `.mjs` due to `ava` not working well in monorepos mixing CommonJS and ESM packages. Once all packages of this monorepos are switched to ESM, this file can be renamed to `.js` and this comment can be removed.